docker/docker-cosmosdb/docker-compose.yml

```diff
@@ -76,3 +76,14 @@ services:
       GW_STATS_ENABLED: "true"
       GW_STATS_PREFIX: "tsdb-gw.stats.dev.tsdbgw_tsdb-gw_1"
       GW_STATS_ADDR: "graphite:2003"
+
+  mt-gateway:
+    hostname: mt-gateway
+    image: grafana/mt-gateway
+    ports:
+      - "6059:6059"
+    environment:
+      MT_GRAPHITE_URL: http://metrictank:6060
+      MT_METRICS_PUBLISH: "true"
+      MT_METRICS_KAFKA_COMP: snappy
+      MT_KAFKA_TCP_ADDR: kafka:9092
```

Comment:
mt-gateway doesn't emit stats like tsdb-gw does?

Reply:
There's some instrumentation, but we don't currently have any configuration for it.

Reply:
why not? should be trivial. just copy over the code from tsdbgw?

Reply:
Hadn't come up yet (the initial goal was minimal functionality).
There's also a mix of graphite and prometheus instrumentation.

I can port it over

Reply:
Added in https://github.com/grafana/metrictank/pull/1645, can rebase and update once that's merged.